README.md

```diff
@@ -289,20 +292,38 @@ Below you can find the list of methods that can be used as expectations:
 
 > Note: if you use such methods as expectations they will not count as expectations, in a test report, for example, but if they fail a clear message will be shown to ease understanding why the test has failed. In the end, test reports are usually useful in cases where tests have failed and we need to understand why. If all tests are green there is nothing to worry about, at least not if they were well implemented.
 
-### Example of a test failure when using such methods as expectations
+### Examples of a test failure when using such methods as expectations
 
 Let's look how a failure would looks like when using some of this methods as expectations.
 
-- Failure when using `waitForElementPresence` as expectation:
+- Failures when using `waitForElementPresence` as expectation:
```

Comment:
Why failures? This should be the example of a single and specific failure

Reply:
I'd say we should have examples for all variations, like 'Failure when using `waitForElementPresence` as expectation and locating element using `by.css`.' Then the same when locating elements using `by.id`, or `by.className`.
Also, we should have examples like this: 'Failure when using `waitForElementPresence` as expectation and locating element using `by.cssContainingText`.' Then the same when locating elements using `by.buttonText`, or `by.partialButtonText`.
What do you think?